### ISSUE TITLE
Can't get "publishAt" working...

### ISSUE THREAD

**Issue by vonmillhausen** [porjo/youtubeuploader#42]
I'm having difficulty getting the "publishAt" setting in the metadata JSON file to do anything using version 18.08; every time I upload a video, it's just showing in my channel as "private", and not as "scheduled".

The command line I'm using is:

`youtubeuploader_linux_armv7 -filename test.mp4 -metaJSON videoMetadata.json -ratelimit 10000 -secrets client_secrets.json`

The contents of videoMetadata.json is taken almost verbatim from your example in the readme:

```
{
  "title": "my test title",
  "description": "my test description",
  "tags": ["test tag1", "test tag2"],
  "privacyStatus": "private",
  "embeddable": true,
  "license": "creativeCommon",
  "publicStatsViewable": true,
  "publishAt": "2018-04-20T09:30:00.000+01:00",
  "categoryId": "10",
  "recordingdate": "2017-05-21",
  "location": {
    "latitude": 48.8584,
    "longitude": 2.2945
  },
  "locationDescription":  "Eiffel Tower",
  "language":  "fr"
}
```

The only differences are that I've set the `publishAt` date to a date that is (currently) in the near future, and I've also removed the `playlistIds` value.

**Comment by vonmillhausen** [comment]
@porjo No worries! I was half afraid I was just doing something dumb :)

**Comment by porjo** [comment]
I've just pushed new release 18.09 which fixes this

**Comment by vonmillhausen** [comment]
@porjo Thanks for the quick work! I just tried 18.09 with the same example as above, but I'm getting a different error this time:

```
Could not read filename file 'videoMetadata.json': parsing time "2018-04-20T09:30:00.000+01:00" as "2006-01-02T15:04:05.000Z": cannot parse "+01:00" as "Z"
Will use command line flags instead
Uploading file 'test.mp4'...
[...]
```

So it seems that the date parsing is off compared to the example you have in the readme. Looking at the code it says it's expecting a date in ISO 8601 format, however the `publishAt` time in my example above _is_ in ISO 8601 format. Also, I'm not sure what the `Could not read filename file` part is about - as you can see from the rest of the error message, it did pull the `publishAt` value out of the videoMetadata.json file, so it's definitely reading its contents OK...

**Comment by porjo** [comment]
Sorry I forgot to update the README (I've done that now). 

Time should be formatted as `2018-04-20T08:30:00.000Z`. I'll make modification in a future version so that the timezone can be specified here instead of having to use UTC.

**Comment by porjo** [comment]
I've published a new release 18.10 which takes timezone. @vonmillhausen you should specify `2018-04-20T09:30:00.000+01:00` now instead of `2018-04-20T08:30:00.000Z`

**Comment by vonmillhausen** [comment]
@porjo Tested this evening - works a treat with the new format, timezone offset works great! Thanks again :)
